source/common/quic/envoy_quic_utils.cc

```diff
@@ -167,7 +167,7 @@ Network::ConnectionSocketPtr
 createConnectionSocket(const Network::Address::InstanceConstSharedPtr& peer_addr,
                        Network::Address::InstanceConstSharedPtr& local_addr,
                        const Network::ConnectionSocket::OptionsSharedPtr& options,
-                       const bool prefer_gro) {
+                       const bool prefer_gro, const bool connect) {
```

Comment:
I suspect this util function is only used in client side. So we can always connect(), but conditionally bind().

Note that if we need to do both, bind() should be done first.

Reply:
Done